spinta/dimensions/param/ufuncs.py

```diff
@@ -1,10 +1,140 @@
-from typing import Any
+from collections.abc import Generator
+from typing import Any, Iterator
 
-from spinta.core.ufuncs import Bind
+from spinta import commands
+from spinta.components import Model
+from spinta.core.ufuncs import Bind, Expr
 from spinta.core.ufuncs import Env
 from spinta.core.ufuncs import ufunc
+from spinta.datasets.components import Param
+from spinta.dimensions.param.components import ParamBuilder, ParamLoader
+from spinta.utils.schema import NotAvailable
+
+
+class _Loop:
+    expr: Expr
+    arg: Any
+
+    def __init__(self, expr, arg):
+        self.arg = arg
+        self.expr = expr
 
 
 @ufunc.resolver(Env, Bind)
 def param(env: Env, bind: Bind) -> Any:
     return env.params[bind.name]
+
+
+@ufunc.resolver(ParamBuilder, Expr)
+def select(env: ParamBuilder, expr: Expr) -> Any:
```

Comment:
Can we change `select` to `read`? I just realized, that `select` is already used and does not have a first argument as model. So to avoid using same name with different semantic, I suggest, to change `select` here to `load`.

The full example would be:

```python
read('Page').loop().more
```

Which would be read into following AST:

```python
getattr(loop(read('Page')), bind('more'))
```

Also you could do this:

```python
read('Page').select(more).loop().more
```

For now we only need `read` ufunc.